common/djangoapps/entitlements/api/v1/urls.py

```diff
@@ -1,11 +1,22 @@
 from django.conf.urls import url, include
 from rest_framework.routers import DefaultRouter
 
-from .views import EntitlementViewSet
+from .views import EntitlementViewSet, EntitlementEnrollmentViewSet
 
 router = DefaultRouter()
 router.register(r'entitlements', EntitlementViewSet, base_name='entitlements')
 
+ENROLLMENTS_VIEW = EntitlementEnrollmentViewSet.as_view({
+    'post': 'create',
+    'delete': 'destroy',
+})
+
+
 urlpatterns = [
     url(r'', include(router.urls)),
+    url(
+        r'entitlements/(?P<uuid>[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12})/enrollments$',
```

Comment:
It might be nice to save this regex to a constant so that we can re-use it elsewhere, much like we do for the COURSE_ID_PATTERN.

Reply:
Agreed.  Can we pull this out into a constant.  Having a reusable UUID would be really helpful